doc/security_need_to_know.md

```diff
@@ -15,19 +15,41 @@ It is commonly stated that a security issue is an issue that:
 
 Given that WASI is a set of Capability-based APIs, all unauthorized actions are not supposed to happen. Most of the above security concerns can be alleviated. What remains for us is to ensure that the execution of Wasm modules is secure. In other words, do not compromise the sandbox. Unless it is explicitly disabled beforehand.
 
-Thus, we share most of the criteria for judging security issues with [the Bytecode Alliance](https://github.com/bytecodealliance/rfcs/blob/main/accepted/what-is-considered-a-security-bug.md#definition).
+### Is this bug considered a security vulnerability?
 
-> [!NOTE]
-> keep updating this document as the project evolves.
+For someone who finds a problem, if a bug causes the program to stop working(CRASH), please treat it as a security problem and report it to a security advisor. The person in charge will look into it and change its category if needed.
```

Comment:
maybe move L24 here:
```markdown
For someone who finds a problem, if a bug causes the program to stop working(CRASH) on tier A support platform/feature(refer to [tierd support document](./tierd_support.md) for more information), please treat it as a security problem and report it to a security advisor. The person in charge will look into it and change its category if needed.
```

Reply:
I am considering ways to lessen the burden on those who report issues. Identifying the platform might be straightforward, but determining which features are involved might not be as simple. Perhaps I should enhance WAMR's issue templates. However, this is good advice.